### PR TITLE
Configure setup py

### PR DESCRIPTION
**Pull request by JamiesonWarner**
I needed to include some extra data files in the setup.py configuration for the package.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import sys
 from setuptools import setup, find_packages
 
-LIBRARY_VERSION = '2.0.1'
+LIBRARY_VERSION = '2.0.3'
```
Comment (ofrancon) [inline setup.py:10]: Why are you skipping `2.0.2`?

File: setup.py [hunk 2]
```diff
@@ -49,11 +49,12 @@ def read(fname):
         PACKAGE_NAME: [
             'countries_regions.csv',
             'countries_regions_phase1_fix.csv',
-            'covid_xprize/examples/predictors/lstm/tests/fixtures/*',
-            'covid_xprize/standard_predictor/data/*',
-            'covid_xprize/standard_predictor/models/*',
-            'covid_xprize/validation/data',
+            'examples/predictors/lstm/tests/fixtures/*',
```
Comment (ofrancon) [inline setup.py:52]: Don't we need the `covid_xprize` folder here? How does it know where to look from the root folder?

Comment (JamiesonWarner) [reply setup.py:52]: It's actually relative to the package. Here the package files are associated with the key `PACKAGE_NAME==covid_xprize` so it is relative to that directory. It took a bit of testing for me to figure it out, but it's also mentioned in the docs; see:

https://docs.python.org/3/distutils/setupscript.html#installing-package-data

File: setup.py (hunk 1 above)
Comment (JamiesonWarner) [reply setup.py:10]: No particular reason; let me change this back...

File: setup.py (hunk 2 above)
Comment (ofrancon) [reply setup.py:52]: ok makes sense. Thanks!

**Review by ofrancon — APPROVED** [review]
lgtm. I guess you can release version `2.0.2` after this merge, unless you have more changes coming.

**Pull request merged**